daemon/loadbalancer.go

```diff
@@ -628,6 +628,31 @@ func (d *Daemon) SyncLBMap() error {
 	return nil
 }
 
+// FrontendList is the list of all k8s service frontends
```

Comment:
Use `k8s: Add LooseMatch() function to validate service presence` as the commit message as `Backport commit` does not mean anything